src/Auth0.AuthenticationApi/AuthenticationApiClient.cs

```diff
@@ -19,7 +19,7 @@ public class AuthenticationApiClient : IAuthenticationApiClient, IDisposable
         static readonly IdTokenValidator idTokenValidator = new IdTokenValidator();
         readonly TimeSpan idTokenValidationLeeway = TimeSpan.FromMinutes(1);
         readonly Uri tokenUri;
-        readonly IAuthenticationConnection connection;        
+        protected readonly IAuthenticationConnection connection;        
```

Comment:
I needed to make this protected in order to be able to create a Test specific class that extends the AuthenticationApiClient to retry on RateLimit Exceptions.